apps/teams-test-app/e2e-test-data/meeting.json

```diff
@@ -340,6 +340,40 @@
       },
       "expectedAlertValue": "updateMicState called with micState: isMicMuted:true, reason:1",
       "expectedTestAppValue": "updateMicState called with micState: isMicMuted:true"
+    },
+    {
+      "title": "joinMeeting API Call - Success",
+      "type": "callResponse",
+      "version": ">2.20.0",
+      "boxSelector": "#box_joinMeeting",
+      "inputValue": {
+        "joinWebUrl": "https://www.example.com/",
+        "subject": "subject",
+        "source": "Other",
+        "chatInfo": {
+          "threadId": "threadId",
+          "messageId": "messageId",
+          "replyChainMessageId": "replyChainMessageId"
+        }
+      },
+      "expectedAlertValue": "joinMeeting called with joinWebUrl: \"https://www.example.com/\"",
+      "expectedTestAppValue": "joinMeeting called with joinWebUrl: \"https://www.example.com/\""
+    },
+    {
+      "title": "joinMeeting API Call - Error",
+      "type": "callResponse",
+      "version": ">2.20.0",
+      "boxSelector": "#box_joinMeeting",
+      "inputValue": {
+        "joinWebUrl": "https://www.example.com/",
+        "subject": "subject",
+        "source": "Other",
+        "chatInfo": {
+          "messageId": "messageId",
+          "replyChainMessageId": "replyChainMessageId"
+        }
+      },
+      "expectedTestAppValue": "Error: Error: JoinUrl or chatInfo not passed"
```

Comment:
@TrevorJoelHarris , I am not sure if this "Error: Error: ..." prompts are what we expect but the test-app-value does match the output.

Reply:
@aditya-darji I'm sure that we have other tests that look for something similar to this, but we shouldn't and I'd like to hold new tests to a higher quality standard. Why does the string "Error" show up twice here?

Reply:
@TrevorJoelHarris @KangxuanYe 
If you notice [MeetingAPIs.tsx](https://github.com/OfficeDev/microsoft-teams-library-js/pull/2151/files#diff-112d449943532744b73c00724f5cc14ad68b6b17088d5f3e2efe22ad146ee3e9:~:text=throw%20new%20Error(%27JoinUrl%20or%20chatInfo%20not%20passed%27)%3B) in this PR, you will notice when we try to validate input, in case the input is incorrect, we `throw new Error('JoinUrl or chatInfo not passed');` in line 424.
This is only used in the e2e test.

When we checked the call stack, we noticed that the calling method has a catch block, where it will append the string "Error: " with the error it caught. Line 70 in [this file](https://github.com/OfficeDev/microsoft-teams-library-js/blob/main/apps/teams-test-app/src/components/utils/ApiWithTextInput.tsx).
Hence we see two errors.

Reply:
Thanks @RishabBindra ! I'll open a bug for someone to look into fixing this for everyone. Appreciate the due diligence.